### DISCUSSION TITLE
MagickScript support

### DISCUSSION THREAD

**Discussion by rlalance** [dlemstra/Magick.NET#788] [Help] [answered]
I saw a few commits that seemed to have removed MagickScript  support.

This is just as I was about to build a backend renderer for MagickScript, can you confirm if it is still in, or removed?

If it is removed, could you point me to the last commit that had MagickScript  support complete?

**Comment by dlemstra** [comment, marked as answer]
MagickScript was removed in version [7.18.0.0](https://github.com/dlemstra/Magick.NET/releases/tag/7.18.0.0) The release before that (7.17.0.1) would be the last commit that included this.

**Reply by rlalance** [reply]
Thank you!
